specification/billing/resource-manager/Microsoft.Billing/preview/2018-08-01-preview/examples/BillingProfileRoleAssignment.json

```diff
@@ -0,0 +1,26 @@
+{
+    "parameters": {
+        "api-version": "2018-08-01-preview",
+        "billingAccountName": "2000100",
+        "billingProfileName": "123",
+        "roleAssignmentId": "{roleAssignmentId}"
+    },
```

Comment:
Missing parameter `billingRoleAssignmentId` and please provide an actual value please instead of {xxx}